### PR TITLE
Added Pypi and Travis-Ci Badges  #6825

### PR DESCRIPTION
**Pull request by vishu1993**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+nereid-activity-stream
+======================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-activity-stream.svg?branch=3.4.0.1)](https://travis-ci.org/openlabs/nereid-activity-stream)
```
Comment (aroraumang) [inline README.md]: This should be for develop branch.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+nereid-activity-stream
+======================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-activity-stream.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-activity-stream)
+[![Downloads](https://pypip.in/download/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Development Status](https://pypip.in/status/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stram/)
+
```
Comment (PritishC) [inline README.md]: Remove these extra lines and leave one blank line between paragraphs.

Coveralls is needed here as it was in the RST file.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+nereid-activity-stream
+======================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-activity-stream.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-activity-stream)
+[![Downloads](https://pypip.in/download/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Development Status](https://pypip.in/status/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stram/)
```
Comment (PritishC) [inline README.md]: stream, not stram.

File: README.rst [hunk 4]
```diff
@@ -1,21 +0,0 @@
...
-
-.. image:: https://coveralls.io/repos/openlabs/nereid-activity-stream/badge.png?branch=develop
-  :target: https://coveralls.io/r/openlabs/nereid-activity-stream
-
-:Authors:
-    `Openlabs Technologies & Consulting (P) Limited
-    <http://www.openlabs.co.in>`_
-
-
-::
```
Comment (PritishC) [inline README.rst:15]: These also have to come in the markdown.

File: README.md [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+[![Downloads](https://pypip.in/download/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Development Status](https://pypip.in/status/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Coverage Status](https://coveralls.io/repos/openlabs/nereid-activity-stream/badge.svg?branch=develop)](https://coveralls.io/r/openlabs/nereid-activity-stream?branch=develop)
+
+
+| Authors        | [Openlabs Technologies & Consulting (P) Limited][]          
+| ------------- |:-------------:
+
+<!-- -->
```
Comment (PritishC) [inline README.md]: Why this HTML comment line?

File: README.md [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+======================
+
+[![Build Status](https://travis-ci.org/openlabs/nereid-activity-stream.svg?branch=develop)](https://travis-ci.org/openlabs/nereid-activity-stream)
+[![Downloads](https://pypip.in/download/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Latest Version](https://pypip.in/version/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Development Status](https://pypip.in/status/trytond_nereid_activity_stream/badge.svg)](https://pypi.python.org/pypi/trytond_nereid_activity_stream/)
+[![Coverage Status](https://coveralls.io/repos/openlabs/nereid-activity-stream/badge.svg?branch=develop)](https://coveralls.io/r/openlabs/nereid-activity-stream?branch=develop)
+
+
+| Authors        | [Openlabs Technologies & Consulting (P) Limited][]
```
Comment (PritishC) [inline README.md:11]: While this gives the resultant table, it doesn't look good in markdown.

Try to beautify it a bit.

Comment (vishu1993) [inline README.md:11]: as discussed it is right

**Comment by PritishC** [comment]
LGTM.

**Pull request merged**